tracer/build/_build/Build.Steps.Debugger.cs

```diff
@@ -51,7 +51,7 @@ partial class Build
         .Requires(() => MonitoringHomeDirectory != null)
         .Executes(() =>
         {
-            DotnetBuild(DebuggerIntegrationTests, framework: Framework);
```

Comment:
We don't want to build for _all_ frameworks here, as we parallelize each of these in CI

Reply:
reverted unrelated clean up :)